core/src/avm2/domain.rs

```diff
@@ -25,6 +25,9 @@ use ruffle_wstr::WStr;
 #[collect(no_drop)]
 pub struct Domain<'gc>(Gc<'gc, DomainData<'gc>>);
 
+/// Weak version of the Domain struct, used to mark children of a Domain when
+/// the egui feature is enabled
+#[expect(dead_code)]
```

Comment:
Shouldn't it be `allow` instead of `expect`? As it won't be dead_code when egui is enabled?